device/iot-device-client/pom.xml

```diff
@@ -82,6 +83,11 @@
             <artifactId>commons-codec</artifactId>
             <version>1.10</version>
         </dependency>
+        <!-- Logging support -->
+        <dependency>
+   			<groupId>org.slf4j</groupId>
+ 			<artifactId>slf4j-api</artifactId>
+    	</dependency>
```

Comment:
This is no breaking change and I really don't like to make changes that introduce an old error once again. Transitioning to logging facades is no enhancement, but a bug fix and has to be done asap. It's probably better to change the CustomLogger or even better completely remove it/exchange it by using slf4j logging facades. This can be done in an hour! If you don't accept the PR this way, I will close it and will open an according issue for it.